src/Homework №1/№17_8-queens_problem_(recursion).py

```diff
@@ -0,0 +1,29 @@
+# task №17: "8-Queens Problem(recursion)"
+BOARD_SIZE = 8
+
+
+def under_attack(col, queens_number):
```

Comment:
here you have 2 empty lines between constant and code, earlier you put 0 lines. Styling should be consistent.